src/devices/machine/z80scc.cpp

```diff
@@ -79,7 +79,7 @@ DONE (x) (p=partly)         NMOS         CMOS       ESCC      EMSCC
 // printf("TAG %lld %s%s Data:%d\n", machine().firstcpu->total_cycles(), __PRETTY_FUNCTION__, m_owner->tag(), data);
 
 #define VERBOSE 0
-#define LOGPRINT(x) do { if (VERBOSE) logerror x; } while (0)
+#define LOGPRINT(x) { do { if (VERBOSE) logerror x; } while (0); }
```

Comment:
Maybe we should provide a logprint.h header with on/off and verbose-level forms so people don't keep reinventing that wheel?